src/main/utils.js

```diff
@@ -182,6 +182,20 @@ function scaleRange(range: Interval, factor: number): Interval {
   return new Interval(start, stop);
 }
 
+function absScaleRange(range: Interval, level: number, halfSpan: number): Interval {
```

Comment:
Can you add some comments to what this function does?

Reply:
Done.

Reply:
Thanks @chfzhang ! I pulled and tested it out and it works great. I am going to squash and merge.